app/src/controllers/testItem/selectors.js

```diff
@@ -297,19 +297,26 @@ export const statisticsLinkSelector = createSelector(
       'filter.eq.hasStats': true,
       'filter.eq.hasChildren': false,
       'filter.in.type': LEVEL_STEP,
+      'filter.has.attributeKey': ownProps.key,
+      'filter.has.attributeValue': ownProps.value,
       'filter.has.compositeAttribute': ownProps.compositeAttribute,
       providerType,
       [providerTypeModifierId]: ownProps[providerTypeModifierId],
       launchesLimit,
       isLatest,
     };
-
     if (ownProps.statuses) {
       params['filter.in.status'] = ownProps.statuses.join(',');
     }
     if (ownProps.types === null) {
       delete params['filter.in.type'];
     }
+    if (ownProps.key === null) {
```

Comment:
No need to do this check, because if the `ownProps.key` or `ownProps.value` will be undefined the parameter will be ommited from the link.
Also please rename the properties to `attributeKey` and `attributeValue` to be more clear.